cmake_format/__main__.py

```diff
@@ -230,6 +245,10 @@ def main():
   mutex.add_argument('-o', '--outfile-path', default=None,
                      help='Where to write the formatted file. '
                           'Default is stdout.')
+
+  mutex.add_argument('-f', '--folder-path', default=None,
+                     help='Folder where should format cmake files.')
```

Comment:
@dkandyba directory where cmake files should be formatted

Reply:
@AByzhynar  Fixed in: 87b6fc709025c5bd8eb8be595494c09cbc6207cc